### PR TITLE
Add missing parsing and documentation for several options

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: man/machinectl.xml [hunk 1]
```diff
@@ -301,6 +301,13 @@
         <literal>,</literal> if another address will be output afterwards. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-q</option></term>
+        <term><option>--quiet</option></term>
+
+        <listitem><para>Suppresses info messages.</para></listitem>
```
Comment (keszybz) [inline man/machinectl.xml]: info → informational?

File: src/udev/udevadm-info.c [hunk 2]
```diff
@@ -250,7 +250,6 @@ static void help(void) {
         printf("%s info [OPTIONS] [DEVPATH|FILE]\n\n"
                "Query sysfs or the udev database.\n\n"
                "  -h --help                   Print this message\n"
-               "  -V --version                Print version of the program\n"
```
Comment (keszybz) [inline src/udev/udevadm-info.c]: This kind of change always requires a commit message!
So this patch removes `udevadm info --version`, but `udevadm --version info` still works.
I'm not sure if this change is warranted: can you provide some justification why that's wanted?

Comment (yuwata) [reply src/udev/udevadm-info.c]: Hmm. Not really wanted. Just for the consistency to other sub-commands. No other sub-commands have `--version` option.

Comment (yuwata) [reply src/udev/udevadm-info.c]: Or should be `--version` added to the other sub-commands?

**Comment by yuwata** [comment]
@keszybz I've force-updated a revised version. For `udevadm info`, I've added commit message to explain why I remove `--version` option. I've added a commit which converts several info to information or informational. Please take a look. Thank you.

File: src/udev/udevadm-info.c (hunk 2 above)
Comment (keszybz) [reply src/udev/udevadm-info.c]: Well... consistency is good, but it shouldn't override everything else. Personally, I think that it's a crappy parser which treats `prog --option verb` different from `prog verb --option`. We *do* have a crappy parser in udevadm, but this change makes things worse. Another consideration is that this breaks backwards compatibility, somebody out there might be using `udevadm info --version` in a script to check udev version.

I'm against removing the option.

**Comment by yuwata** [comment]
@keszybs I've force-pushed a revised version. In the revised version, I've added --version option to all `udevadm` subcommands, instead of removing the option from `udevadm info`. I've added a deprecate message in `udevadm hwdb` command.

**Comment by poettering** [comment]
CI fialure:

```
../src/udev/udevadm-control.c: In function ‘adm_control’:
../src/udev/udevadm-control.c:158:25: error: implicit declaration of function ‘print_version’; did you mean ‘print_help’? [-Werror=implicit-function-declaration]
                         print_version();
                         ^~~~~~~~~~~~~
```

**Comment by yuwata** [comment]
@poettering Updated. I am sorry for many mistakes today...

**Comment by keszybz** [comment]
Thanks for working on this. Those docs were really in a need of update.

**Comment by keszybz** [comment]
`KVM: entry failed, hardware error 0x0` and ICE — looks unrelated.

**Pull request merged**